oracle/config/api.go

```diff
@@ -51,7 +61,7 @@ func (c *APIConfig) ValidateBasic() error {
 		return fmt.Errorf("provider interval, timeout and reconnect timeout must be strictly positive")
 	}
 
-	if len(c.URL) == 0 {
+	if len(c.URL) == 0 && len(c.Endpoints) == 0 {
```

Comment:
the error message for this will now not be accurate.  Only mentions provider url